datanode/gateway/graphql/schema.graphql

```diff
@@ -618,7 +618,7 @@ type Query {
     "ID for an order"
     id: ID!
 
-    "Version of the order (omitted or 0 for most recent; 1 for original; 2 for first amendment, etc)"
+    "Version of the order (omitted for most recent; 1 for original; 2 for first amendment, etc)"
```

Comment:
Approved but ... question. The protos says the following, and that doesn't seem exactly the same as this description does: 

> Version for the order, initial value is version 1 and is incremented after each successful amend

Is the protos unspecific about "most recent"?

Reply:
I would say the difference is that the protos don't mention the fact that omitting the version would give us the most recent, but as for "1 for original", I believe it boils down to the same thing - the initial value, or the first version, before amending it. 
The phrase "2 for first amendment, etc" could be clearer, for sure. However, I believe it essentially boils down to "...and is incremented after each successful amend", meaning every time we update the order, the version will be incremented, thus after version 1 (initial/original) we have version 2, 3, 4...

Reply:
I think if I replace it with something like:

"Order version number, starting at 1 for the original order and incrementing by 1 for each successful amendment (omitted for the most recent version)."

Reply:
@zale144 Thank you, that sounds good. Sorry, hadn't realised they weren't the same thing.